polar-core/src/resource_block.rs

```diff
@@ -54,7 +54,7 @@ fn validate_parsed_declaration((name, term): (Term, Term)) -> PolarResult<Parsed
 
         ("roles", Value::Dictionary(_)) | ("permissions", Value::Dictionary(_)) => {
             let msg = format!("Expected '{}' declaration to be a list of strings; found a dictionary", name.to_polar());
-            Err(ValidationError::ResourceBlock { msg, term}.into())
```

Comment:
how'd this happen with format check 😕

Reply:
Not sure; `rustfmt` seems pretty apathetic about how I format that line